### ISSUE TITLE
Tracking issue: SvelteKit prototype

### ISSUE THREAD

**Issue by fkling** [sourcegraph/sourcegraph#55027]
This is a "live" issue to keep track of SvelteKit prototype related tasks. Everybody is welcome to pick up any of the UI issues, to get a feel for how developing with SvelteKit is like. I'm happy to help and answer any questions!

### Missing UI/UX
- [x] File and folder table view for repo root and folder page
  This is a simple change that requires to use the existing data and render it as a table instead of a list.
  - #55616
- [x] Dropdown menu component (e.g. for user menu)
  Basically porting the [Wildcard](https://storybook.sgdev.org/?path=/story/wildcard-menu--menu-example) menu component to Svelte
- [ ] Pagination for commits branches and tags page
  ~Either infinity scroll or pagination, whatever the current app uses.~ If pagination then the same component that the contributors page uses could be used, or we could properly port the PageSelector or PageSwitcher Wildcard components.
  - https://github.com/sourcegraph/sourcegraph/pull/58847
- [ ] Revision picker
- ~[ ] Repository settings pages
  Requires adding new routes and "gating" them to users who have permissions to change settings.~
- [x] Use new search query input
  Requires UI updates and making the completion extension extensible so that a new Svelte-based dropdown can be used.
  - https://github.com/sourcegraph/sourcegraph/pull/58244
- [x] Make search input available on repository page
  Add the search input to the header and add keyboard shortcut handling.
- [ ] Add symbols sidebar
- [x] Add support for missing search result types
  - https://github.com/sourcegraph/sourcegraph/pull/55542
- [ ] Missing blob view features
  - [x] Token navigation + hovercards: Requires changes to allow specifying a different hovercard implementation. Also a lot of the log depends on the `blobProps` facet which should be improved somehow.
    - #58559 
  - [ ] Blame view: Also requires changes to allow different UI components
  - [ ] Reference panel
  - [ ] Ownership panel
  Once [the history panel PR]() is merged, add another panel for ownership information (just like in the current web app). A decision has to be made whether to load ownership data in the corresponding `load` function (probably not?), or when the component is created.
- [ ] Make bottom panel resizable. Resizeable panel logic already exists but it's not a good solution. It should be rewritten and support vertical resizing.
- [x] Set appropriate page title for every page.
  - https://github.com/sourcegraph/sourcegraph/pull/58807
- [x] Make individual search results clickable. The React version uses a complex solution to make text selection and clicking work. For the prototype we should keep it simple and just support clicking (e.g. via a simple anchor element).
  - https://github.com/sourcegraph/sourcegraph/pull/59004
- [ ] "Scroll line into view" doesn't work reliably because the current solution assumes that the scroll container is inside the editor itself. In the prototype the whole page is scrollable instead. Supporting this requires to change the existing CodeMirror extension that provides this feature.
- [x] Add a way to quickly switch between the React version and Svelte version.
  - https://github.com/sourcegraph/sourcegraph/pull/58782
- [x] Show recent commit info in folder file list
  - https://github.com/sourcegraph/sourcegraph/pull/58865
- [x] Search results null state
  - https://github.com/sourcegraph/sourcegraph/pull/59029

**Comment by peterguy** [comment]
Pretty sure this has been deprecated in favor of the project board, right @fkling?

**Comment by fkling** [comment]
Closing this as it has been superseded by the project board.
